From af0bea5fdad98552aafb4f6a23becddc5c3326fe Mon Sep 17 00:00:00 2001
From: Bredfen <53024173+Bredfen@users.noreply.github.com>
Date: Thu, 18 Jul 2019 02:59:59 +0300
Subject: [PATCH] Initial commit

---
 README.md | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..db9683a
--- /dev/null
+++ b/README.md
@@ -0,0 +1 @@
+# Java2.First
\ No newline at end of file
-- 
2.21.0.windows.1

